Header header
bool valid
sensor_decoder/LaneMsg left_lane
sensor_decoder/LaneMsg right_lane
int32 n_next_lanes
sensor_decoder/LaneMsg[] next_lanes
int32 n_obstacles
sensor_decoder/ObstacleMsg[] obstacles
float64 x
float64 y
float64 v
float64 ax
float64 ay
float64 theta
float64 omega
